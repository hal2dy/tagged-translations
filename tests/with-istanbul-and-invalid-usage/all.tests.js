runTests(
  __dirname,
  [],
  ['babel-plugin-istanbul', 'babel-plugin-macros'],
  transform => {
    expect(transform).toThrowErrorMatchingSnapshot()
  }
)
