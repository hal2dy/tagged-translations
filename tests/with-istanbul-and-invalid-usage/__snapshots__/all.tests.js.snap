// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test with-istanbul-and-invalid-usage.js 1`] = `"with-istanbul-and-invalid-usage.js: ../../src/macro: tagged-translations/macro can only be used in tagged template expression. You tried SequenceExpression."`;
